test-support/pom.xml

```diff
@@ -43,6 +43,12 @@
             <version>${testcontainers.version}</version>
         </dependency>
 
+        <dependency>
```

Comment:
if it already in the parent pom we don't need it here

Reply:
I removed it from the parent pom since